Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -10,8 +10,16 @@ public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
 
 /// An `UIView` subclass that shows summary and preview information about a given channel.
 open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, ThemeProvider, SwiftUIRepresentable {
+    /// The content of this view.
+    public struct Content<ExtraData: ExtraDataTypes> {
```

Comment:
Does this have to be generic? 🤔 I think you can access the `ExtraData` of the enclosing type.